$OpenBSD: patch-CMakeLists_txt,v 1.4 2014/04/18 09:30:48 brad Exp $

Don't confuse scripts who want the version with appended 'svn' goo.

--- CMakeLists.txt.orig	Sun Mar  2 21:57:43 2014
+++ CMakeLists.txt	Sun Mar  2 22:13:02 2014
@@ -29,7 +29,7 @@ set(LLVM_VERSION_MAJOR 3)
 set(LLVM_VERSION_MINOR 5)
 
 if (NOT PACKAGE_VERSION)
-  set(PACKAGE_VERSION "${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}svn")
+  set(PACKAGE_VERSION "${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}")
 endif()
 
 option(LLVM_INSTALL_TOOLCHAIN_ONLY "Only include toolchain files in the 'install' target." OFF)
